shell/platform/embedder/embedder.cc

```diff
@@ -761,12 +775,18 @@ static sk_sp<SkSurface> MakeSkSurfaceFromBackingStore(
 
   SkSurfaceProps surface_properties(0, kUnknown_SkPixelGeometry);
 
+  std::optional<SkColorType> color_type =
+      FlutterFormatToSkColorType(texture->format);
+  if (!color_type) {
+    return nullptr;
+  }
+
   auto surface = SkSurfaces::WrapBackendTexture(
       context,                      // context
       backend_texture,              // back-end texture
       kBottomLeft_GrSurfaceOrigin,  // surface origin
       1,                            // sample count
-      kN32_SkColorType,             // color type
```

Comment:
`kN32_SkColorType` is either `BGRA8` or `RGBA8` depending on the machine that builds the engine. This caused Skia to reject RGBA8 textures if the engine was built on a machine where BGRA8 is the native color format.

Reply:
Oh wow - nice catch.